R/get_attributes.R

```diff
@@ -12,13 +12,14 @@
 #' @export
 #' @importFrom dplyr bind_rows
 #' @examples
+#' options("emld_db" = "eml-2.1.1")
 #' f <- system.file("tests", options("emld_db"), 
```

Comment:
We really should remove all mentions of `options("emld_db")` and replace with `emld::eml_version()` throughout the source code....

(you may already know that in RStudio, `ctrl`+`shift`+`F` is a great way to find all occurrences of a pattern across all files in a project)

Reply:
not sure what to do with the instances in `data-raw/construct_complexTypes.R`

otherwise I'm about to push the other instances, including changes to the readme and a vignette